src/winetricks

```diff
@@ -11069,6 +11069,51 @@ load_xact()
 
 #----------------------------------------------------------------
 
+if test "$W_ARCH" = "win64"; then
+    w_metadata xact_x64 dlls \
+        title="MS XACT Engine (64-bit only)" \
+        publisher="Microsoft" \
+        year="2010" \
+        media="download" \
+        file1="../directx9/directx_Jun2010_redist.exe" \
+        installed_file1="$W_SYSTEM64_DLLS_WIN64/xactengine2_0.dll"
+
+    load_xact_x64()
+    {
+        w_package_warn_win64 # https://bugs.winehq.org/show_bug.cgi?id=41618#c5
```

Comment:
This should instead be:
```
w_package_unsupported_win64 # to block 32-bit prefixes
    if w_workaround_wine_bug 41618; then
        w_warn "While this helps some games, it completely breaks others. You've been warned."
    fi
```

`w_package_warn_win64` is intended for verbs that work on win32/win64, but have issues on win64.

Reply:
Done